Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: all oc-create-build oc-start-build
```

Comment:
Doesn't seem related to graphing but OK

Reply:
Heh, I can drop it, I was just using it for local builds, but you're right :)

Reply:
Keep it, seems useful!

Reply:
I just moved it to a separate PR #116